scripts/custom_targeting_to_coverage.R

```diff
@@ -0,0 +1,58 @@
+##this script creates a coverage table (proportion of target population vaccinated at admin 0 level) based on information inside 
```

Comment:
Same comment as above (in create_custom_targeting_table.R) about the script name and saved input / output file names.